src/libs/sea_ai/src/AIBalls.cpp

```diff
@@ -434,6 +428,27 @@ uint64_t AIBalls::ProcessMessage(MESSAGE &message)
     return 0;
 }
 
+void BALL_PARAMS::Save(CSaveLoad *pSL)
```

Comment:
@np-c0mp1ete why this 'Save' is needed? Cannot we just define pointer as an uint64_t instead?

Reply:
it is needed because I changed the type of `sBallEvent` from `char[]` to `std::string`. Also, this way it's explicit what we want to serialize and how.